### PR TITLE
Backport to DuckDB v1.0.0

### PR DESCRIPTION
**Pull request by lmangani**
Thanks @carlopi for the guidance and assistance 🦆

### CODE REVIEW

**Comment by carlopi** [comment]
Error is connected to conversation at https://github.com/lmangani/duckdb-extension-clickhouse-sql/issues, where this fix works only when building vs 1.0.0. I can figure it out the proper version of this and send a PR, or a workaround is pointing in .github/workflows/MainDistributionPipeline.yml to have duckdb_version as 1.0.0.

File: .github/workflows/MainDistributionPipeline.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
     name: Build extension binaries
     uses: duckdb/extension-ci-tools/.github/workflows/_extension_distribution.yml@main
     with:
-      duckdb_version: main
+      duckdb_version: 1.0.0
```
Comment (carlopi) [inline .github/workflows/MainDistributionPipeline.yml:19]:
```suggestion
      duckdb_version: v1.0.0
```

File: .github/workflows/MainDistributionPipeline.yml [hunk 2]
```diff
@@ -25,7 +25,7 @@ jobs:
     uses: duckdb/extension-ci-tools/.github/workflows/_extension_deploy.yml@main
     secrets: inherit
     with:
-      duckdb_version: main
+      duckdb_version: 1.0.0
```
Comment (carlopi) [inline .github/workflows/MainDistributionPipeline.yml:28]:
```suggestion
      duckdb_version: v1.0.0
```

**Review by carlopi — COMMENTED** [review]
Maybe this one?

**Comment by lmangani** [comment]
> Maybe this one?

Thanks for keeping an eye on my rookie mistakes 😉

**Comment by lmangani** [comment]
well... not going as smooth as I was hoping this morning :) any suggestion or perhaps a little PR to help us out of this spin?

**Comment by carlopi** [comment]
I will send a PR later / tomorrow

**Comment by carlopi** [comment]
The problem in this PR has been fixed in the extension-ci-tools repository, in this https://github.com/duckdb/extension-ci-tools/pull/44.

Possibly closing and reopening the PR is the easiest way to re-trigger CI while having it point to the new workflow (just restarting might not be enough).

I think that if you retrigger it should work now correctly.

**Comment by carlopi** [comment]
I need to look at this then, back in a few minutes.

**Pull request closed without merging**